native-extension-examples/klarna-checkout/README.md

```diff
@@ -0,0 +1,126 @@
+
+In the custom component configuration, enter the extension ID and a component identifier corresponding to the mapping done in klarna-checkout.module.ts. Hit save.
+
+![configure experience](../../documentation/assets/configure_experience.png)
+
+**8\. Payment Configuration**
+
+Before the component can be used, you will need to configure a gateway or payment method with a compatible klarna script and assign it to the division associated with the app.
+
+A postman collection to quickly setup one of these should be accessible via the [SAP-samples Github](https://github.com/SAP-samples/upscale-commerce-open-payment-integration). Instructions on how to install a payment configuration using a postman collection can be found [here](https://github.com/SAP-samples/upscale-commerce-open-payment-integration/tree/main/postman/klarna/custom-checkout).
+
+Please also be sure to fill in your config's gatewayProviderName (labelled as "Payment gateway name") as the string specified in klarna-checkout.component.ts above.
+
+![configure payment](../../documentation/assets/configure_payment.png)
+
+## Testing
+If all the above setup is complete, please follow the steps below to test your new custom component!
+
+**1\. Download the app from workbench**
+
+**2\. Extract the contents of the zip**
+
+**3\. Install dependencies:**
+
+Access the project root in terminal and run "npm install"
+
+**4\. Add the library module to your application:**
+
+Note: If you're using version 0.59.0 or higher, this step is automated and can be skipped.
+
+Open the file src/app/core/extension.module.ts in your ide/text editor
+
+Add the following to the first list of imports:
+
+    import { UpscaleExtensionModule } from 'klarna-checkout';
+
+Then add UpscaleExtensionModule to the array of native extension modules:
+
+    // Add Native Extension Modules here
+    const modules = [ UpscaleExtensionModule ];
+
+**5\. Start app:**
+
+Access the project root in terminal and run "npm start"
+
+**6\. Test in browser:**
+
+Access the application via localhost:4200, add a product to cart, and proceed to checkout. A klarna payment form should appear in the place of the regular checkout!
+
+![test in pwa](../../documentation/assets/test_in_pwa.png)
+
+## How it all works
+The Klarna Checkout component makes use of the aforementioned front end services made available by upscale-provided dependencies; upscale-web-storefront-sdk and caas-service-client-angular, as well as some provided by angular itself.
+
+Refer to [Klarna Checkout component](https://github.com/SAP-samples/upscale-commerce-open-payment-integration/blob/main/native-extension/klarna/checkout/How%20Klarna%20Checkout%20Component%20Works.docx) for details.
+
+## License
+Copyright (c) 2021 SAP SE or an SAP affiliate company. All rights reserved. This project is licensed under the Apache Software License, version 2.0 except as noted otherwise in the [LICENSE](LICENSES/Apache-2.0.txt) file.
```

Comment:
We's in the future I guess.

```suggestion
Copyright (c) 2022 SAP SE or an SAP affiliate company. All rights reserved. This project is licensed under the Apache Software License, version 2.0 except as noted otherwise in the [LICENSE](LICENSES/Apache-2.0.txt) file.
```